### ISSUE TITLE
tmux configuration error on tmux 2.9a

### ISSUE THREAD

**Issue by Corea** [wookayin/dotfiles#15]
## Problem
The error is raised when making a new tmux session.

```
.tmux.conf:201: invalid option: window-status-activity-attr
.tmux.conf:211: invalid option: pane-border-fg
.tmux.conf:212: invalid option: pane-border-bg
.tmux.conf:213: invalid option: pane-active-border-fg
.tmux.conf:214: invalid option: pane-active-border-bg
```

## Related Documents
- https://github.com/tmux/tmux/issues/1689
- https://github.com/tmux/tmux/wiki/FAQ#how-do-i-translate--fg--bg-and--attr-options-into--style-options

## Solution
- use `*-style` instead of these options.
  - use `window-status-activity-style` instead of `window-status-activity-attr`
  - use `pane-border-style` instead of `pane-border-fg` and `pane-border-bg`
  - use `pane-active-border-style` instead of `pane-active-border-fg` and `pane-active-border-bg`

**Comment by wookayin** [comment]
Tmux has so many breaking changes without any deprecation. Are you aware of any configuration that would work with both <2.8 and >=2.9?

**Comment by wookayin** [comment]
Changes in 3.0 is even more crazy: https://raw.githubusercontent.com/tmux/tmux/3.0/CHANGES

**Comment by Corea** [comment]
I didn't get other errors except this with 2.9a, so I guess the less part of settings could be used for 2.9.

**Comment by wookayin** [comment]
This has been fixed in dc3be82, thanks! The setting should work both in tmux 2.4 and 2.9a.
